### ISSUE TITLE
Lack of variants

### ISSUE THREAD

**Issue by SinaKuhestani** [MirageDigitals/Logo#1]
A good logo needs these variants:
-white and black version
-transparent background
-ico version
-svg version
-png version

**16x16 pixels**

Commonly used favicon dimensions for most browsers (standard favicon size)

**32x32 pixels**

Best size for favicon used for taskbar shortcut icons; favicon dimension for macOS; favicon resolution for retina-based smart devices

**48x48 pixels**

Favicon size used for Chrome browser

**76x76 pixels**

Home screen fav icon on iPad

**96x96 pixels**

Favicon image size for desktop shortcut icons; can be used for Chrome

**120x120 pixels**

Typical ICO size for iPhone

**128x128 pixels**

Favicon size for Chrome web store

**144x144 pixels**

Favicon icon size used for Chrome browser

**152x152 pixels**

Common favicon dimensions for iPad

**167x167 pixels**

Favicon pixel size on iPad Retina

**180x180 pixels**

Mostly used size of a favicon for Apple touch icons and iPhone Retina

**192x192 pixels**

Size recommendation for a web app manifest by Google Developers

**196x196 pixels**

Chrome home screen icon on Android

**228x228 pixels**

Size for favicon for Opera Coast

**256x256 pixels**

Possible favicon dimensions for Chrome

**300x300 pixels**

Favicon dimension required by Squarespace

**384x384 pixels**

Fav icons size for Chrome browser

**512x512 pixels**

Favicon sizes which WordPress requires

I suggest to use a 1024x1024 canvas.

**Comment by SinaKuhestani** [comment]
@farnaznic  please add this variants to release the first version (1.0.0). Your final design is eye catching. Just make the requested variants.

**Comment by mitramj** [comment]
@farnaznic  I agree @SinaKuhestani . We usually need all of these variants in front-end development.
